pywbem_mock/_cimrepository.py

```diff
@@ -2127,23 +2158,21 @@ def _get_associated_instancenames(self, inst_name, namespace, assoc_class,
     #
     ####################################################################
 
-    def ReferenceNames(self, **params):
+    def ReferenceNames(self, ObjectName, ResultClass=None, Role=None):
         """
-        Implements a mock WBEM server responder for
+        Implements a CIM repository responder for
         :meth:`~pywbem.WBEMConnection.ReferenceNames`
         """
 
         namespace = self.namespace
```

Comment:
See previous comment about `namespace = self.namespace`. Only this time, the namespace is (optionally) passed in ObjectName.